src/test/java/com/wikia/webdriver/testcases/desktop/navigation/global/GlobalNavigationSearching.java

```diff
@@ -22,6 +24,14 @@
                           {"mediawiki119", "szl", "Mix", "Specjalna:Szukaj"}};
   }
 
+  // TODO: Remove SZL DataProvider & SZL tests when not needed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)